scripts/batch-score.coffee

```diff
@@ -0,0 +1,50 @@
+# Description:
+#   Script for batch wise score.
+#
+# Commands:
+#   hubot score bxx
+# 
+# Author:
+#   aseem09
+
+cron = require('node-cron')
+module.exports = (robot) ->
+
+  listOfUsersWithCount = (year) ->
+  
+    sorted = []
+    for own key, user of robot.brain.data.users
+
+      temp2=user[4]
+      user_year=temp2.split('')
```

Comment:
Are you sure it will be consistent with your data?

Reply:
I have tested this by hard coding temp2 as <digit><alphabet><alphabet> eg. "1st"